wai-extra/wai-extra.cabal

```diff
@@ -1,5 +1,5 @@
 Name:                wai-extra
-Version:             3.0.32
+Version:             3.1.0
```

Comment:
Please include a lower bound on base for 4.10 here to prevent untested GHC versions from being used.

Reply:
Done. I cherry-picked 38a828a500ae0bc5e0f6210e0cdb3f2cdeb0d31d.

Reply:
That's not what I meant. I still want @kazu-yamamoto's sign-off before introducing those lower bounds on the rest of the packages here

Reply:
Ok, just for `wai-extra` now.